Assets/LeapMotion/Scripts/Utils/Utils.cs

```diff
@@ -430,6 +430,46 @@ public struct ChildrenEnumerator {
       capsule.height = (capsuleSpaceDistToA + capsule.radius) * 2;
     }
 
+    /// <summary>
+    /// Recursively searches the hierarchy of the argument GameObject to find all of the
+    /// Colliders that are attached to the object's Rigidbody (or that _would_ be 
+    /// attached to its Rigidbody if it doesn't have one) and adds them to the provided
+    /// colliders list. Warning: The provided "colliders" List will be cleared before
+    /// use.
+    /// 
+    /// Colliders that are the children of other Rigidbody elements beneath the argument
+    /// object are ignored.
+    /// </summary>
+    public static void FindColliders<T>(GameObject obj, ref List<T> colliders) where T : Collider {
```

Comment:
I think that it should be obvious enough from the docs of the method.  The ref keyword also might be misleading for some, implying that you might allocate a new list inside the method and assign it to be the new colliders list.